.travis.yml

```diff
@@ -32,20 +32,20 @@ matrix:
       script:
         - ./functional.sh verbose
     - stage: build
-      name: "Unit Tests: Python 2.7 with Attrs 18 and 19"
+      name: "Unit Tests: Python 2.7 with Attrs 18 to 21"
       python: "2.7"
       script:
-        - tox -e py27-attrs18,py27-flake8,py27-coverage
+        - tox -e py27-attrs18,py27-attrs19,py27-attrs20,py27-attrs21,py27-flake8,py27-coverage
     - stage: build
       name: "Unit Tests: Python 3.5 with and without PyInotify; Pytest 5.1 and 5.3"
       python: "3.5"
       script:
         - tox -e py35,py35-pyinotify,py35-pytest51,py35-pytest53,coverage
     - stage: build
-      name: "Unit Tests: Python 3.7 with Attrs 18 and 19; PyTest 4.6 and 5.3"
```

Comment:
I realized there was a bug here, the comment said we were testing with attrs 19 but we weren't. I've fixed it and included 2 more versions of attrs.